sycl/test-e2e/ESIMD/lsc/lsc_slm_atomic_smoke.cpp

```diff
@@ -149,10 +149,10 @@ bool test(queue q) {
                 // arg0 and arg1 must provide values which guarantee the loop
                 // is not endless:
                 for (auto old_val = lsc_slm_atomic_update<op, T>(
-                         offsets, new_val, exp_val, m);
+                         offsets, exp_val, new_val, m);
                      any(old_val < exp_val, !m);
-                     old_val = lsc_slm_atomic_update<op, T>(offsets, new_val,
-                                                            exp_val, m))
+                     old_val = lsc_slm_atomic_update<op, T>(offsets, exp_val,
```

Comment:
lsc_slm_atomic_update() is clearly under-documented it only says that the functions has the first and the second operands. It doesn't tell anything about the expected operands order for cmpxchg.
Please add some extra comment there explaining how the meanings of arguments for cmpxchg.